src/compute/src/compute_state.rs

```diff
@@ -294,7 +294,10 @@ impl<'a, A: Allocate + 'static> ActiveComputeState<'a, A> {
             None => {}
         }
         match enable_chunked_stack {
-            Some(setting) => crate::containers::stack::use_chunked_stack(setting),
+            Some(setting) => {
+                info!(setting, "Using chunked stack");
```

Comment:
The message is a bit misleading, because `setting` is a `bool` and can be `false`, in which case we would _not_ use the chunked stack.

Reply:
Changed to something neutral. Thanks!